homeworks/12-mars-mobx/src/components/Photo/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from "react";
+
+import { ReactComponent as HeartIcon } from "assets/icons/heart.svg";
+import { Photo as PhotoType } from "types/Photo";
+import { Figure, Image, FigureCaption, FavouriteButton } from "./Photo.styled";
+
+type Props = {
```

Comment:
Please use interface for component prop definition. Also a name like `Props` gives little to no value.

The standard way of naming props interface is `<ComponentName>Props`